src/components/presentational/DateRangeNavigator/DateRangeNavigator.css

```diff
@@ -7,11 +7,16 @@
     text-align: center;
     font-weight: bold;
     border-bottom: solid 1px var(--mdhui-border-color-0);
-    position: sticky;
-    position: -webkit-sticky;
-    top: env(safe-area-inset-top, 0px);
     background: var(--mdhui-background-color-0);
     z-index: 1;
+    position: relative;
```

Comment:
Can you explain this line? It seems unnecessary to me.

Reply:
The left/right icons are absolutely positioned.  When I removed position:sticky I needed to add position:relative in order for them to continue to be absolutely positioned to this element.

Reply:
Makes sense.